src/pages/AddItems/AddItems.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import { db } from '../../lib/firebase';
+import { useSnackbar } from 'notistack';
+import filter from '../../lib/filter';
+import {
+  TextField,
+  FormControl,
+  FormLabel,
+  FormControlLabel,
+  RadioGroup,
+  Radio,
+  Button,
+} from '@material-ui/core';
+import { StyledForm } from './elements';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [itemFreq, setItemFreq] = useState(7);
+
+  const { enqueueSnackbar } = useSnackbar();
+
+  const updateGroceryItem = (event) => {
+    setGroceryItem(event.target.value);
+  };
+
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: itemFreq,
+      purchaseDates: [],
+      purchaseEstimates: [itemFreq],
+    };
+
+    const filtered = filter(props.list, groceryItem, true);
+
+    if (filtered.length > 0) {
+      enqueueSnackbar('This item is already on your list', {
+        variant: 'error',
+      });
+    } else {
+      db.collection(props.userToken).add({ formData });
+      setGroceryItem('');
+      setItemFreq(7);
+      enqueueSnackbar(`${groceryItem} successfully added`, {
+        variant: 'success',
+      });
+    }
+  };
+
+  const radioBtnHandler = (event) => {
+    setItemFreq(+event.target.value);
+  };
+
+  return (
+    <div>
+      <StyledForm id="add-item-form" onSubmit={submitGroceryItem}>
+        <TextField
+          id="item"
+          label="Item name:"
```

Comment:
@ablizben this is the subtle label for the text field that I was talking about

Reply:
Thank you!